### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by dorkerdev** [justinfrankel/licecap#126]
Seems like the original devs have jumped ship.

**Comment by esaruoho** [comment]
a build was made on 8th of june 2022. is that so bad?
from their site:
```
[LICEcap v1.32 for Windows](https://www.cockos.com/licecap/licecap132-install.exe) (Jun 8 2022) (250kb installer)
[LICEcap v1.32 for macOS](https://www.cockos.com/licecap/licecap132.dmg) (Jun 8 2022) (876kb DMG)
```

**Comment by dorkerdev** [comment]
@esaruoho No, but I was mainly referring to all of the open issues and the lack of contributor responses.

**Comment by esaruoho** [comment]
> @esaruoho No, but I was mainly referring to all of the open issues and the lack of contributor responses.

i get what you're saying. taking a look at the issues here, there's plenty of "yeah dude mac doesn't work" from years and years, and the original creators of the tickets aren't closing them. ideally, the `/issues` here would be a tightly run ship. people might get confused into thinking that macOS doesn't work, if all they see are floods of  screen recording issues or big sur etc issues.

**Comment by AlexJ-invision** [comment]
It requires a bit of configuration (as described in some of the issues) but once you've done that the tool works perfectly.

**Comment by nkantar** [comment]
According to the website, this is just a mirror.
